src/components/file/README.md

```diff
@@ -10,7 +10,7 @@ npm install --save @ecl/file
 
 - **"id"** (string) (default: random string)
 - **"icon"** (object) (default: {}): object of type Icon; file type
-- **"variant"** (string) (default: '') Thumbnail
+- **"variant"** (string) (default: '') Name of the variant [default, thumbnail]
```

Comment:
why mentioning "default" as a variant..? default is no variant, basically

Reply:
actually, in the template the docs are like this:   - "variant" (string) (default: 'default') Name of the variant [default, thumbnail]. I updated it here